Week2/Assignment4/src/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Namaste React Bootcamp | Assignment 4</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="root">
+      <!-- Some error occurred -->
```

Comment:
Just curious, why you added here a comment?

Reply:
Ah , its not necessary i just added in case if react fails to update the root then one way we could see this error is through this comment.